assets/styles/parallax-text-type1.scss

```diff
@@ -6,11 +6,21 @@
   .parllax-text-name {
     position: absolute;
     left: -10%;
-    font-size: 60px;
+    @include respond-to(sm) {
```

Comment:
mobile first! you gotta set generic styles for mobile and then go with the break points